Run Python tests on CI

Currently, running tests for the Python modules in the CI environment fails with the following error:

```
ImportError while importing test module '[...]/kfp-operator/argo/kfp-compiler/acceptance/test_compiler_acceptance.py'.
...
    from _lzma import *
E   ModuleNotFoundError: No module named '_lzma'
```
This prevents us from running these tests as part of the `make test-all` target.

--- a/Makefile
+++ b/Makefile
@@ -95,7 +95,7 @@ test-argo:
 	$(MAKE) -C argo/kfp-sdk test
 	$(MAKE) -C argo/events test
 
-test-all: test helm-test # test-argo See https://github.com/sky-uk/kfp-operator/issues/54
+test-all: test helm-test test-argo
 	$(MAKE) -C argo/events test
 
 ##@ Build

